### PR TITLE
APPS:2952 Codecov report is being shown again on README

### PR DESCRIPTION
**Pull request by oleksander-kiiashko**
## PR Description

Ticket: https://spryker.atlassian.net/browse/APPS-2952

### CODE REVIEW

**Comment by dereuromark** [comment]
Please also see https://github.com/spryker/propel-encryption-behavior/issues/4 - maybe it could be added as well.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+*.phar binary
+*.ttf binary
+*.woff binary
+*.woff2 binary
+*.eot binary
+*.ico binary
+*.mo binary
+*.pdf binary
+*.xsd binary
+*.exe binary
```
Comment (RuslanKhudenko) [inline .gitattributes:22]: Noticed we have more files in suite-nonsplit/registry-service/etc:
*.xcf binary
*.eps binary
*.psd binary
*.dll binary
*.doc binary
*.docx binary

Just information think about it

**Review by matweew — APPROVED** [review]

**Review by RuslanKhudenko — APPROVED** [review]

File: .gitattributes (hunk 1 above)
Comment (oleksander-kiiashko) [reply .gitattributes:22]: I have used standard list of files after creation via spryk

Comment (dereuromark) [reply .gitattributes:22]: Those are only relevant as per repo
So if none of those are used here, we are good.

**Comment by dereuromark** [comment]
Nice work!

**Pull request merged**